src/org/labkey/test/components/SetDefaultPageClass.java

```diff
@@ -0,0 +1,19 @@
+package org.labkey.test.components;
+
+import org.labkey.test.Locator;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+
+public class SetDefaultPageClass extends LabKeyPage
+{
+    public SetDefaultPageClass(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public void selectDefaultGrid(String gridName)
+    {
+        click(Locator.linkWithText("select").withAttributeContaining("onclick",gridName));
+        clickAndWait(Locator.tagWithText("span","Done"));
```

Comment:
Isn't the "Done" a button? I would have expected this to use one of the click button and wait helpers.